cedar-policy-core/src/ast/request.rs

```diff
@@ -350,6 +350,14 @@ impl From<Context> for PartialValue {
     }
 }
 
+impl From<PartialValue> for Context {
+    fn from(value: PartialValue) -> Self {
+        Self {
+            context: value.into(),
+        }
+    }
+}
```

Comment:
Should this fail if the `PartialValue` is not a record or unknown?

Reply:
Removed it.